WooCommerce/WooCommerceUITests/Tests/StatsTests.swift

```diff
@@ -15,12 +14,14 @@ final class StatsTests: XCTestCase {
         try LoginFlow.logInWithWPcom()
     }
 
-    func skipped_test_load_stats_screen() throws {
+    func test_load_stats_screen() throws {
         try TabNavComponent().goToMyStoreScreen()
-    }
-
-    func skipTillImplemented(file: StaticString = #file, line: UInt = #line) throws {
-        try XCTSkipIf(true,
-            "Skipping until test is properly implemented", file: file, line: line)
+            .verifyTodayStatsLoaded()
```

Comment:
This might look redundant (and it most probably is), but I had an idea of also adding a function that selects the `Today` tab, and calling it before checking the `Today` stats.

Reply:
i'm leaving this as is too, i think not adding that extra tap ensures that upon signing in the merchant would land on the expected tab (Today) instead of other tabs